### DISCUSSION TITLE
expireOrdersAfter: Minute does not work

### DISCUSSION THREAD

**Discussion by mayankoker** [saleor/saleor#15263] [General]
I have set up this value in minute .
mutation UpdateOrderSettingsForChannel($id: ID!, $orderSettings: OrderSettingsInput!) {
  channelUpdate(id: $id, input: {orderSettings: $orderSettings}) {
    channel {
      id
      orderSettings {
        expireOrdersAfter
      }
    }
  }
}

{
  "id": "Q2hhbm5lbDoy",
  "orderSettings": {
    "expireOrdersAfter": 1
  }
}

But orders are not getting expired.

**Comment by mayankoker** [comment]
This worked but even I have set "expireOrdersAfter": 1
 ( 1 minute as per documentation ) 
But orders expired randomly after 1.30 hrs 

Can some one explain what could be reason , or there is some bug at Saleor ?

**Comment by tomaszlud** [comment]
Perhaps the other discussion will help you: https://github.com/saleor/saleor/issues/14352
